### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MingjiaLiu1995**
Update README file per developer's feedback from validation

### CODE REVIEW

File: Excel-HelloWorld-TaskPane-JS/README.md [hunk 1]
```diff
@@ -23,13 +23,19 @@ https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
...
+
+1. **Download the sample code**
+
+   You can use either of the ways to download the sample code:
+   * In the `Office Add-ins Development Kit` extension, find the sample in `Sample gallery`, click the "create" button at the top of the sample page.
+   * Clone or download this sample to a folder on your computer, then open the folder in Visual Studio Code.
+   
+1. **Open the Office Add-in Development Kit**
+   
+   Click the <img src="./assets/Icon_Office_Add-ins_Development_Kit.png" width="30"/> button in the side panel to open the extension.
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/README.md:38]: Use the same alt text in this image as the icon in the extension displays.
```suggestion
   Click the <img src="./assets/Icon_Office_Add-ins_Development_Kit.png" width="30" alt="Office Add-in Dev Kit"/> button in the side panel to open the extension.
```

File: Excel-HelloWorld-TaskPane-JS/README.md [hunk 2]
```diff
@@ -23,13 +23,19 @@ https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
...
-
 1. **Install the Office Add-ins Development Kit**
     
-    Install the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) extension in Visual Studio Code marketplace. Click the <img src="./assets/Icon_Office_Add-ins_Development_Kit.png" width="30"/> button in the side panel to open the extension.
+    Install the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) extension in Visual Studio Code marketplace. 
+
+1. **Download the sample code**
+
+   You can use either of the ways to download the sample code:
+   * In the `Office Add-ins Development Kit` extension, find the sample in `Sample gallery`, click the "create" button at the top of the sample page.
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/README.md:33]: I think this describes the UI flow better. Also, UI options should be **bold** and we should favor "select" over "click" to support non-mouse inputs.
```suggestion
   * Open the Office Add-ins Development Kit extension. Select **View Samples** and choose a sample from the **Sample Gallery**. Select the **Create** button in the top-right corner of the sample page.
```

File: Excel-HelloWorld-TaskPane-JS/README.md [hunk 3]
```diff
@@ -23,13 +23,19 @@ https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
...
+   * Clone or download this sample to a folder on your computer, then open the folder in Visual Studio Code.
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/README.md:34]:
```suggestion
   * [Clone](https://docs.github.com/repositories/creating-and-managing-repositories/cloning-a-repository) this repository or download this sample to a folder on your computer. Then, open the folder in Visual Studio Code.
```

**Review by AlexJerabek — CHANGES_REQUESTED** [review]

File: Excel-HelloWorld-TaskPane-JS/README.md (hunk 1 above)
Comment (MingjiaLiu1995) [reply Excel-HelloWorld-TaskPane-JS/README.md]: Add the alt text

File: Excel-HelloWorld-TaskPane-JS/README.md (hunk 2 above)
Comment (MingjiaLiu1995) [reply Excel-HelloWorld-TaskPane-JS/README.md]: I change the description as below:
Open the Office Add-ins Development Kit extension and view samples in the **Sample gallery**. Select the **Create** button in the top-right corner of the sample page.

The reason is that developers may see this page under different circumstances. For developers that already arrive at sample page in VSC from web link or sample gallery, they only need to create, so we might not emphasize on previous actions like Select View samples and choose a sample; For developers that starts from GitHub, they can still go to download and view samples in the sample gallery.

**Comment by MingjiaLiu1995** [comment]
@hermanwenhe This change also needs to be merged after you merge #85 , then we apply the pattern to all samples and add Alex as reviewer for a final review. Thanks!

**Review by AlexJerabek — APPROVED** [review]

**Review by hermanwenhe — APPROVED** [review]

**Pull request merged**